### ISSUE TITLE
Segfault in theme::countdown::location

### ISSUE THREAD

**Issue by jostephd** [wesnoth/wesnoth#3599]
There is a segfault as soon as I start the second scenario of the UMC campaign "Bad Moon Rising":

```
$ ./wesnoth-debug -cBad_Moon_Rising_I --campaign-scenario=01_Abandoned --campaign-difficulty=2
Thread 1 "wesnoth-debug" received signal SIGSEGV, Segmentation fault.
0x00005555569804a7 in theme::countdown::location (this=0x555558153a80, screen=...) at src/theme.cpp:468
468                     return display::get_singleton()->get_theme().
```

`get_status_item("battery")` returns nullptr.

It goes away if I revert e58e57d1d01ebfe0b8fe91cdf08124f738ee992c.

**Comment by jostephd** [comment]
I can also reproduce this without UMC, by starting a new local MP game, clicking "I'm ready" and immediately pressing Esc.

**Comment by gfgtdf** [comment]
that c++ code should just check the return value of get_status_item before using it, maybe falling back to the other case (`status_item::location(screen)`) when it returns nullptr.

**Comment by jyrkive** [comment]
Fixed in commit 4a23c8bed18d280754f9f4f8ea35486fe7ad48c5.
